src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
+        Colors color = Colors.values()[index];
+
+        return String.valueOf(color);
```

Comment:
Instead of using `String.valueOf(color)`, consider using `color.name()` to get the string representation of the enum constant. This approach is recommended to avoid potential issues if the `toString()` method is overridden in the future.